lib/pure/algorithm.nim

```diff
@@ -49,18 +47,18 @@ type
     Descending, Ascending
 
 proc `*`*(x: int, order: SortOrder): int {.inline.} =
-  ## Flips ``x`` if ``order == Descending``.
-  ## If ``order == Ascending`` then ``x`` is returned.
+  ## Flip `x` if `order == Descending`.
```

Comment:
`Flips` is preferred.

Reply:
The [contributing guideline](https://nim-lang.github.io/Nim/contributing.html#documentation) says:
```
The preferred documentation style is to begin with a capital letter and use the imperative (command) form.
```